test/caching-precompiler.js

```diff
@@ -145,7 +146,7 @@ test('uses babelConfig for babel options when babelConfig is an object', functio
 	t.true('inputSourceMap' in options);
 	t.false(options.babelrc);
 	t.same(options.presets, ['stage-2', 'es2015']);
-	t.same(options.plugins, [customPlugin, powerAssert, rewrite, transformRuntime]);
+	t.same(options.plugins, [customPlugin, powerAssert, throwsHelper, rewrite, transformRuntime]);
```

Comment:
> we don't enhance t.throws with power-assert.

Right. `t.throws` is [listed as NON_ENHANCED_PATTERNS here](https://github.com/sindresorhus/ava/blob/84c05fe98ee5887e49262d48f1134cdf19118a7f/lib/enhance-assert.js#L23-L29). power-assert doesn't touch its arguments.